app/src/main/java/com/codesoom/assignment/service/TodoService.java

```diff
@@ -0,0 +1,47 @@
+package com.codesoom.assignment.service;
+
+import com.codesoom.assignment.common.exception.TaskNotFoundException;
+import com.codesoom.assignment.dto.TaskDto;
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.repository.TodoRepository;
+
+import java.util.List;
+
+/**
+ * {@link Task}
+ * Controller에서 요청들을 받아서 Task에 대한 저장, 수정, 삭제, 조회 등의 작업을 수행한다.
+ */
```

Comment:
`Controller`가 아니라 누가 호출하더라도 TodoService는 이 역할을 할 수 있어야 합니다.

```suggestion
/**
 * Task에 대한 저장, 수정, 삭제, 조회 등의 작업을 수행한다.
 */
```